core/src/test/resources/dataset/data_set.json

```diff
@@ -41,6 +41,15 @@
       }
     }
   ],
+  "dataInputPeriods": [
```

Comment:
Backend model allows to create dataInputPeriods with null openingDate and closingDate. Currently it fails when parsing the object. It should be easily replicated with a test.
Make these two properties nullable.

Reply:
Ok, I tought they was mandatory. I'll fix it.

Reply:
Fixed. 

Thanks for your feedback!